### PR TITLE
Modified appliance_time_and_locale_configuration

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
changed the python class to new style

### Issues Resolved
None
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG file.

### CODE REVIEW

File: hpeOneView/resources/settings/appliance_time_and_locale_configuration.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 from hpeOneView.resources.resource import ResourceClient
 
 
-class ApplianceTimeAndLocaleConfiguration(object):
+class ApplianceTimeAndLocaleConfiguration(ResourceClient):
```
Comment (VenkateshRavula) [inline hpeOneView/resources/settings/appliance_time_and_locale_configuration.py:30]: The base class should be Resource class

File: examples/appliance_time_and_locale_configuration.py [hunk 2]
```diff
@@ -27,38 +27,42 @@
...
+time_and_locale = time_and_locale.data
 # Set to use appliance local time and date server
 time_and_locale['ntpServers'] = ['127.0.0.1']
 # Set locale to Chinese (China) with charset UTF-8
 time_and_locale['locale'] = 'zh_CN.UTF-8'
 # Remove the date and time, we do not want to update it manually
 time_and_locale.pop('dateTime')
-time_and_locale = oneview_client.appliance_time_and_locale_configuration.update(time_and_locale)
+time_and_locale = time_and_locales.create(data=time_and_locale)
 print("\n## Updated appliance time and locale configurations successfully!")
```
Comment (VenkateshRavula) [inline examples/appliance_time_and_locale_configuration.py:61]: There is no update operation for this resource. So change the print statement as below.
```
print("\n## Changed the appliance time and locale configurations successfully!")
```

**Review by VenkateshRavula — CHANGES_REQUESTED** [review]
Update changelog and endpoint-support files with this resource. Also modify oneview_client.py file and test_oneview_client.py files respectively.

Refer the below PR - https://github.com/HewlettPackard/oneview-python/pull/28

**Review by VenkateshRavula — APPROVED** [review]
Please modify endpoints file

File: endpoints-support.md [hunk 3]
```diff
@@ -27,6 +27,9 @@
...
 |<sub>/rest/appliance/trap-destinations/{id}</sub>                                        |PUT       |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/appliance/trap-destinations/{id}</sub>                                        |DELETE    |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|    **Appliance Time and Locale Configuration**
+|<sub>/rest/appliance/configuration/time-locale</sub>                                     |GET       |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:31]: keep white_check_mark only for api 2400

**Review by yuvirani — APPROVED** [review]

**Pull request merged**